runtime/vm/jnimisc.cpp

```diff
@@ -478,6 +478,7 @@ findClass(JNIEnv *env, const char *name)
 		J9ClassLoader *classLoader = getCurrentClassLoader(currentThread);
 		U_8 *trimmedName = verifiedName;
 		/* Allow class names in the form LName; since JDK does, despite it being expressly against the spec */
+		// TODO: andrewc Should QName also be accepted? I suspect it is also against spec
```

Comment:
this one we'll skip for now for the same reason as above. Currently its unclear how qtypes are surfaced in the API